frontend/src/components/userPage/UserPageContent.jsx

```diff
@@ -130,6 +141,40 @@ const UserPageContent = ({ user }) => {
         }
     }
 
+    const handlePatchCellphone = async () => {
+        if (String(cellphoneNumber).length != 10) {
+            alert('Please input a valid cellphone number')
+        } else {
+            try {
+                await axios
+                    .patch(`${url}/users/cellphone`, {
+                        id: user.id,
+                        cellphoneNumber: String(cellphoneNumber),
+                    })
+                    .then(function (response) {
+                        updateUser(response.data)
+                    })
+            } catch (error) {
+                console.error('Error changing location:', error)
+            }
+        }
+    }
+
+    const handleDeleteCellphone = async () => {
+        try {
+            await axios
+                .patch(`${url}/users/cellphone`, {
+                    id: user.id,
+                    cellphoneNumber: null,
+                })
+                .then(function (response) {
+                    updateUser(response.data)
+                })
+        } catch (error) {
+            console.error('Error changing location:', error)
+        }
+    }
```

Comment:
This can be merged with handlePatchCellphone. It's basically doing the same thing, just the cellphone is null.

Reply:
Done